### PR TITLE
Objects Automated Test - LPS-141661 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-141661

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase [hunk 1]
```diff
@@ -903,13 +903,95 @@ definition {
 	}
 
 	@description = "LPS-135401 - Verify that it is possible to delete a relation with an entry on the Relationship tab"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanDeleteRelationOnRelationshipTab {
-		property portal.acceptance = "true";
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:909]: Hi @andreefarias ,

You don't need to delete this line. 

Thanks

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path [hunk 2]
```diff
@@ -262,7 +262,7 @@
 	</tr>
 	<tr>
 		<td>OBJECT_RELATIONSHIPS_TAB</td>
-		<td>//a[contains(text(),'Relationships')]</td>
+		<td>//a[contains(text(),'Relationship')]</td>
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path:265]: Hi @andreefarias 

You don't need to change this line, let me know why you changed it.

Thanks!

**Review by evanilson-santana — CHANGES_REQUESTED** [review]
Hi @andreefarias ,

Please, check my comments and let me know if you have any questions.

Thanks!!!

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path (hunk 2 above)
Comment (andreefarias) [reply modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path:265]: There are test cases that look for "Relationship". I believe it gets more generic this way.

Comment (evanilson-santana) [reply modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path:265]: Okay.

**Review by evanilson-santana — APPROVED** [review]

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**